File: PHP/1794.php
<?php
function valida($pecas, $valores){
    if($valores[0] <= $pecas && $pecas <= $valores[1]){
        return True;
    } else {
        return False;
    }
}

$pecas = trim(fgets(STDIN));
$lavar = explode(' ', trim(fgets(STDIN)));
$secar = explode(' ', trim(fgets(STDIN)));

if(valida($pecas, $lavar) && valida($pecas, $secar)){
    printf("possivel\n");
} else {
    printf("impossivel\n");
}
?>

File: Python/1794.py
pecas = int(input())
min_l, max_l = map(int, input().split())
min_s, max_s = map(int, input().split())

if ((min_l <= pecas <= max_l) and (min_s <= pecas <= max_s)):
    print("possivel")
else:
    print("impossivel")
